ci_tutorial/packages_pipeline/single_configuration.rst

```diff
@@ -0,0 +1,102 @@
+Package pipeline: single configuration
+======================================
+
+We will start with the most simple case, in which we only had to build 1 configuration, and that configuration 
+can be built in the current CI machine.
+
+As we described before while presenting the different server binary repositories, the idea is that package builds
+will use by default the ``develop`` repo only, which is considered the stable one for developer and CI jobs.
+
+Let's make sure we start from a clean state:
```

Comment:
Maybe at this point it would be good to point to the configuration instructions or remind our users that they should have cloned the examples2 repo that has the .conanrc, to prevent someone that starts here and deletes the entire cache and remotes?